_config.yml

```diff
@@ -18,18 +18,44 @@
 # You can create any custom.scss variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 
+# meta
 title: Sheffield Hackspace
+description: Sheffield Hackspace
+
+# URLs
+## the subpath of your site, e.g. /blog
+baseurl: ""
+## the base hostname & protocol for your site, e.g. http://example.com
+url: "https://www.sheffieldhackspace.org.uk"
+logo: /assets/images/logo.svg
+rss_feed: /feed.xml
+goat_counter_url: "https://sheffieldhackspacewebsite.goatcounter.com"
+
+# socials
 email: trustees@sheffieldhackspace.org.uk
-description: >- # this means to ignore newlines until "baseurl:"
-  Sheffield Hackspace
-baseurl: "" # the subpath of your site, e.g. /blog
-url: "https://www.sheffieldhackspace.org.uk" # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: "@shhmakers"
 github_username:  "sheffieldhackspace"
 facebook_username: "SHHMakers"
 instagram_username: "SHHMakers"
-logo: /assets/images/logo.svg
-goat_counter_url: "https://sheffieldhackspacewebsite.goatcounter.com"
+
+# information - mainly used in spaceapi.json
+address: "Sheffield Hackspace, Portland Works, Randall St, Highfield, Sheffield S2 4SJ"
+latitude: 53.369933
+longitude: -1.473812
+timezone: "Europe/London"
+
+# membership
+membership_plans:
+  - name: Normal membership
+    value: 6
```

Comment:
Could we pull these into the joining page? Doesn't need to be complex. Just so if someone goes to update prices they wont miss one place or the other.